.github/ISSUE_TEMPLATE/release.md

```diff
@@ -116,3 +116,12 @@ release-time. To initialize a benchmark run for each production runtime
 * Commit the changes to your branch and push to the remote branch for review.
 * The weights should be (Currently manually) checked to make sure there are no
     big outliers (i.e., twice or half the weight).
+
+### Security Audit
+
+Before each release, we should do `Security Audit`
+
+* Go to [Security Audit](https://github.com/Manta-Network/Manta/actions/workflows/audit.yml).
+* Open `Run workflow` drop-down menu.
+* Choose your branch and run the workflow.
+* Finnaly, an audit report will be generated.
```

Comment:
```suggestion
* An audit report will be generated
* Address any reported findings before release
```
What do we do with reported audit findings?
Can we say we want to have 0 violations on each release (after excluding the deps)?

Reply:
I think we should file an issue, and not sure every vulnerable crate has a fix.